docs/basic-usage.md

```diff
@@ -63,8 +63,29 @@ Similarly, the traditional `MANIFEST.in` file is replaced by the `tool.poetry.re
 `tool.poetry.exclude` sections. `tool.poetry.exclude` is additionally implicitly populated by your `.gitignore`. For
 full documentation on the project format, see the [pyproject section]({{< relref "pyproject" >}}) of the documentation.
 
+### Setting a Python Version
+
+{{% note %}}
+Unlike with other packages, Poetry will not automatically install a python interpreter for you.
+If you want to run Python files in your package like a script or application, you must _bring your own_ python interpreter to run them.
+{{% /note %}}
+
 Poetry will require you to explicitly specify what versions of Python you intend to support, and its universal locking
 will guarantee that your project is installable (and all dependencies claim support for) all supported Python versions.
+Again, it's important to remember that---unlike other dependencies---setting a Python version is merely specifying which versions of Python you intend to support.
```

Comment:
You could combine your above revision with with this one more concisely by saying,

"Critically, setting a Python version is merely specifying which versions of Python you intend to _support._ To run your application or library, you must _bring your own_ python interpreter (for instance using `pyenv`)."

That would make it more concise.

Reply:
For now, I think it would be good to have an explicit note in a callout box. I agree that your phrasing is more concise, but I think this will be a bit more visible.